platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/MemoryIndexDiskKeyValueStore.java

```diff
@@ -65,15 +64,6 @@ public class MemoryIndexDiskKeyValueStore<D> implements AutoCloseable, Snapshota
      */
     private final String storeName;
 
-    /** The minimum key that is valid for this store. Set in {@link MemoryIndexDiskKeyValueStore#startWriting} to
-     * be reused in {@link MemoryIndexDiskKeyValueStore#endWriting()}
-     */
-    private final AtomicLong minValidKey;
-    /** The maximum key that is valid for this store.  Set in {@link MemoryIndexDiskKeyValueStore#startWriting} to
-     * be reused in {@link MemoryIndexDiskKeyValueStore#endWriting()}
-     */
-    private final AtomicLong maxValidKey;
```

Comment:
These fields are not needed. The corresponding values can be read from the index.